packages/dialect-react-ui/components/Avatar/index.tsx

```diff
@@ -34,7 +33,7 @@ export default function Avatar({ publicKey, size = 'regular' }: PropTypes) {
     >
       <div className={`${textStyleMap[size]}`}>
         {program?.provider.connection ? (
-            <AddressImage
+            <CardinalAvatar
               height="100%"
```

Comment:
Same with the `height, width` props — to be consistent on styles approach — I thinks it's better to pass className prop to underlying component, and use tailwind styles here, like className="dt-h-full"